scripts/autostartliota

```diff
@@ -0,0 +1,32 @@
+#! /bin/sh
+### BEGIN INIT INFO
+# Provides:          Autostarts liota.
+# Required-Start:    $all
+# Required-Stop:
+# Default-Start:     2 3 4 5
+# Default-Stop:
+# Short-Description: Starts liotad.py in background at reboot.
+### END INIT INFO
+
+PATH=/sbin:/usr/sbin:/bin:/usr/bin
+
+do_start() {
+	sudo python /etc/liota/packages/liotad.py &
+}
+
+case "$1" in
```

Comment:
How the re-loading of loaded packages is taken care, in case of restart scenario?